pkgs/tools/package-management/comma/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, nix-index
+, fzy
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "comma";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "nix-community";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-IT7zlcM1Oh4sWeCJ1m4NkteuajPxTnNo1tbitG0eqlg=";
+  };
+
+  patches = [
+    (fetchpatch {
+      name = "support-nix-2.4.patch";
+      url = "https://github.com/nix-community/comma/commit/ca8998f4377afcab5904b29ea88fe329716535c8.patch";
+      sha256 = "sha256-e8ic2swpUNLEQ/uBQx9vD7FEBw6aloZ5mPexcsf9oFw=";
+    })
+  ];
```

Comment:
not needed anymore as i merged PRs in the comma repo. i also tagged a new release

Reply:
Fixed :+1: